app/BUILD.bazel

```diff
@@ -500,6 +520,47 @@ android_library(
     visibility = ["//visibility:public"],
 )
 
+kt_android_library(
+    name = "app",
+    custom_package = "org.oppia.app.ui",
+    srcs = APP_FILES,
+    enable_data_binding = 1,
+    resource_files = glob(LAYOUTS),
```

Comment:
Isn't this duplicating resource files with ``databinding_resources``?

Reply:
Yes, just removed it and everything builds perfectly!